lib/Maintenance/Tasks/CleanupBrickTablesTask.php

```diff
@@ -30,13 +30,23 @@ class CleanupBrickTablesTask implements TaskInterface
      * @var LoggerInterface
      */
     private $logger;
+    private $mapLowerToCamelCase = [];
 
     /**
      * @param LoggerInterface $logger
      */
     public function __construct(LoggerInterface $logger)
     {
         $this->logger = $logger;
+        
+        $obPath = PIMCORE_CLASS_DIRECTORY . '/DataObject/Objectbrick/Data';
```

Comment:
Should this be a constant?

Reply:
Yes, defined in Bootstrap.php:
\Pimcore\Bootstrap::defineConstants()
```
...
        $resolveConstant('PIMCORE_CLASS_DIRECTORY', PIMCORE_PRIVATE_VAR . '/classes');
...
```

Reply:
I mean, if we should define `PIMCORE_CLASS_DIRECTORY . '/DataObject/Objectbrick/Data'` as a class constant instead of a variable in the method.

Reply:
Updated PR to use class constant